toolkit/tools/internal/retry/retry.go

```diff
@@ -8,6 +8,14 @@ import (
 	"time"
 )
 
+const (
```

Comment:
This works for me, but if we start doing more of these maybe we can move to a dedicated config struct like:
```go
type RetryConfig {
    BackoffBase float
    RetryAttempts int
    RetryDuration  time.Duration
}

const DefaultDownloadConfig = RetryConfig{ 2.0, 5, time.Second}
```
(note: Just threw that together on a whim, not sure its valid)

Reply:
(to be clear, current version is fine)